auth.go

```diff
@@ -145,6 +153,16 @@ func (s *slackAuth) OnAuth(fn func(*slack.OAuthResponse)) {
 	s.callback = fn
 }
 
+func (s *slackAuth) ConfigureSlackButton(options *SlackButtonOptions) error {
```

Comment:
i prefer the slack button config on the creation parameters, just to not have to configure more stuff later